kong/conf_loader.lua

```diff
@@ -70,6 +70,10 @@ local DYNAMIC_KEY_NAMESPACES = {
     injected_conf_name = "nginx_admin_directives",
     prefix = "nginx_admin_", -- TODO: nginx_http_admin (optional)
   },
+  {
+    injected_conf_name = "nginx_status_directives",
+    prefix = "nginx_status_",
```

Comment:
Note that the "correct" prefix for this new category of injected directives should be `nginx_http_status_` (note the TODO comments in the previous prefixes). It was unfortunate that we weren't more specific in the naming when we first introduced them, but we haven't updated their name yet for backwards-compatibility reasons.

We should either directly introduce this new prefix with the chosen future naming convention, or also include a TODO comment for a future refactor of the directives injection prefixes. The latter might be preferable for now. This isn't a blocker, but it's important to keep it in mind.

Reply:
Thank you for catching this!
Changed to `nginx_http_status_`